### PR TITLE
Console improvements and fixes

### PR DESCRIPTION
**Pull request by sesposito**
Add support for deleting a leaderboard record.
Improve config storage objects file handling.

### CODE REVIEW

File: console/ui/src/app/leaderboard/records/records.component.html [hunk 1]
```diff
@@ -20,18 +20,19 @@ <h6 class="mr-2 d-inline font-weight-bold">An error occurred: {{error}}</h6>
...
       <tr>
-        <th style="width: 315px">Owner ID</th>
+        <th style="width: 380px">Owner ID</th>
         <th>Username</th>
         <th style="width: 150px">Score</th>
         <th style="width: 150px">Subscore</th>
         <th style="width: 150px">Rank</th>
         <th style="width: 150px">No. of scores</th>
         <th style="width: 180px">Expiry Time</th>
+        <th style="width: 180px" *ngIf="deleteAllowed()">Remove</th>
```
Comment (mofirouz) [inline console/ui/src/app/leaderboard/records/records.component.html:30]: Please use 90px here to match everywhere else.

File: console/ui/src/app/config/config.component.ts [hunk 2]
```diff
@@ -131,6 +131,53 @@ export class ConfigComponent implements OnInit, OnDestroy {
     }
   }
 
+  // If the file is in JSON, it stringifies the value of the storage object.
+  private processFile(fIn: File, callback: (f: File) => void): void {
```
Comment (mofirouz) [inline console/ui/src/app/config/config.component.ts:135]: Whilst this is really good, I'd suggest that we make this much simpler - i.e. make the server code such that it only accepts `value` as a JSON object. This is indeed a breaking change to the behaviour of this API from Nakama2 but I think it's worth it to make it much much easier to deal with the data.

File: server/console_leaderboard.go [hunk 3]
```diff
@@ -107,8 +107,8 @@ func (s *ConsoleServer) ListLeaderboardRecords(ctx context.Context, in *api.List
 			return nil, status.Error(codes.InvalidArgument, "Invalid limit - limit must be between 1 and 100.")
 		}
 		limit = in.GetLimit()
-	} else if len(in.GetOwnerIds()) == 0 || in.GetCursor() != "" {
-		limit = &wrappers.Int32Value{Value: 1}
+	} else if len(in.GetOwnerIds()) == 0 || in.GetCursor() == "" {
+		limit = &wrappers.Int32Value{Value: 10}
```
Comment (mofirouz) [inline server/console_leaderboard.go:111]: why this change?

File: server/api_leaderboard.go [hunk 4]
```diff
@@ -114,8 +114,8 @@ func (s *ApiServer) ListLeaderboardRecords(ctx context.Context, in *api.ListLead
 			return nil, status.Error(codes.InvalidArgument, "Invalid limit - limit must be between 1 and 100.")
 		}
 		limit = in.GetLimit()
-	} else if len(in.GetOwnerIds()) == 0 || in.GetCursor() != "" {
-		limit = &wrappers.Int32Value{Value: 1}
+	} else if len(in.GetOwnerIds()) == 0 || in.GetCursor() == "" {
+		limit = &wrappers.Int32Value{Value: 10}
```
Comment (mofirouz) [inline server/api_leaderboard.go:118]: why this change? Should we set the limit value in the console request?

**Review by mofirouz — CHANGES_REQUESTED** [review]
Some changes required.

File: server/api_leaderboard.go (hunk 4 above)
Comment (sesposito) [reply server/api_leaderboard.go]: Discussed IRL

File: server/console_leaderboard.go (hunk 3 above)
Comment (sesposito) [reply server/console_leaderboard.go]: Discussed IRL

File: console/ui/src/app/config/config.component.ts (hunk 2 above)
Comment (sesposito) [reply console/ui/src/app/config/config.component.ts]: I've made the change server side instead.

**Review by mofirouz — APPROVED** [review]
